src/FlarumPWAServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of askvortsov/flarum-pwa
+ *
+ *  Copyright (c) 2021 Alexander Skvortsov.
+ *
+ *  For detailed copyright and license information, please view the
+ *  LICENSE file that was distributed with this source code.
+ */
+
+namespace Askvortsov\FlarumPWA;
+
+use Flarum\Foundation\AbstractServiceProvider;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Kreait\Firebase\Contract\Messaging as FirebaseMessagingContract;
+use Kreait\Firebase\Factory as FirebaseFactory;
+
+class FlarumPWAServiceProvider extends AbstractServiceProvider
+{
+    public function register()
+    {
+        $this->container->bind(FirebaseMessagingContract::class, function ($container) {
+            $settings = $container[SettingsRepositoryInterface::class];
+
+            return (new FirebaseFactory)
+                ->withServiceAccount(json_decode($settings->get('askvortsov-pwa.firebaseConfig'), true))
```

Comment:
Should we try/catch this and re-raise so that if the content is corrupted, the error is easier to find? Also, shouldn't running this be conditional on the config being present?